### ISSUE TITLE
Import custom radio stations in .pls and .m3u format to Volumio 2

### ISSUE THREAD

**Issue by heis2201** [volumio/Volumio2#334]
Often, people have their favorite audio streams setup as .pls or .m3u files which is well supported in Volumio 1.5. I am hoping for a function in Volumio 2 enabling the import of .m3u and .pls audio stream files. Either adding them to the existing web radios, handling them as playlists, or just playing them as if they were audio files...

**Comment by apinner** [comment]
Related to https://github.com/volumio/Volumio2/issues/332

**Comment by heis2201** [comment]
So, that means that I can simply copy my "old" .pls and .m3u files to somewhere (/var/lib/mpd/music/webradio ?)and they will appear in the player?

**Comment by volumio** [comment]
Yes, I guess this is the idea... And it's a good one so I'll implement it...

**Comment by heis2201** [comment]
Great! Thanks!

**Comment by apinner** [comment]
@volumio what happened to the idea of the volumio selection of stations?

**Comment by volumio** [comment]
Still on todo list... I've also created an API server based on parse... Do you think we could use it?

**Comment by apinner** [comment]
I'm not familiar with Parse. Isn't it shutting down?

**Comment by volumio** [comment]
They open sourced it... 
https://github.com/ParsePlatform/parse-server

Any alternatives? Or API server (ideally json based) you're familiar with?

**Comment by aidanreilly** [comment]
would be really nice to see this in.

**Comment by Cheddoleum** [comment]
Interestingly it does read .m3u files and include them after a scan of a music folder; and you do get the list of queue/play options -- it just won't actually play the URLs in the file.

**Comment by heis2201** [comment]
Any update or recipe on how to do this best?

**Comment by buzink** [comment]
+1

**Comment by buzink** [comment]
Better support for custom radio streams would be nice. I think it would be most transparant to simply have a place somewhere in the browse tab where you can paste any url. Volumio could convert streams to the json playlist format and save them. Later you could even add support through plugins for youtube, soundcloud and podcast urls. All through the same add-a-url interface.

**Comment by heis2201** [comment]
Dear all,
this issue is nearing its 6th birthday soon. Should we close it considering that there is manual way to achieve an import of m3u radio playlists (https://github.com/suuuehgi/volumio2m3u)?
